testing/BUILD.gn

```diff
@@ -6,6 +6,10 @@ import("//flutter/common/config.gni")
 import("//flutter/shell/config.gni")
 import("testing.gni")
 
+config("dynamic_symbols") {
+  ldflags = [ "-rdynamic" ]
```

Comment:
`if (is_clang && is_linux)` as this does not make sense on Windows.

Reply:
done